overrides/main.html

```diff
@@ -0,0 +1,2 @@
+<html lang="tr">
+{% extends "base.html" %}
```

Comment:
I need to fork and test this.

Reply:
Can we remove this; the final rendition results in

```
<html lang="tr">
<!DOCTYPE html>
<html lang="en">
```

which is invalid markup.

If you want, you can override `base.html`.